packages/mercury-case-analysis/README.md

```diff
@@ -0,0 +1,81 @@
+# mercury-case-analysis
+
+Server-side library for analyzing technical Q&A cases (prompt + expected response).
+
+Outputs include:
+- Relevance metrics comparing the original prompt to prompts generated from the expected response
+- Quality ratings for the prompt/response pair
+- Optional suggested rewrites
+
+## Install
+
+```bash
+npm install mercury-case-analysis
+```
+
+Requires Node 18+.
+
+## Quick start
+
+```ts
+import { makeAnalyzeCases } from "mercury-case-analysis";
+import { createOpenAI, wrapLanguageModel } from "mongodb-rag-core/aiSdk";
+import { BraintrustMiddleware } from "mongodb-rag-core/braintrust";
```

Comment:
note that we dont publish mongodb-rag-core anymore. if you plan on publishign this package to npm, consider reexporting mongodb-rag-core from here too